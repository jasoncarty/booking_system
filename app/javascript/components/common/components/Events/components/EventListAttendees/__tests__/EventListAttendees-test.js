import React from 'react';
import { create } from 'react-test-renderer';

import EventListAttendee from './../index';

const defaultProps = {
  isReserves: true,
  attendees: [
    {
      user: {
        name: 'jason carty',
        email: 'jason@email.com'
      }
    }
  ],
  eventId: 1
};

describe('<EventListAttendee />', () => {
  it('matches snapshot when isReserves', () => {
    const element = create(<EventListAttendee {...defaultProps} />);
    expect(element).toMatchSnapshot();
  });

  it('matches snapshot when !isReserves', () => {
    const props = {
      ...defaultProps,
      isReserves: false
    };
    const element = create(<EventListAttendee {...props} />);
    expect(element).toMatchSnapshot();
  });
});
