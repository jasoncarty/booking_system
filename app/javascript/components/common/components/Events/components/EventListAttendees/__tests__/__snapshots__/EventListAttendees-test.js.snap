// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventListAttendee /> matches snapshot when !isReserves 1`] = `
<div
  className="event-attendees not-reserves"
>
  <div
    className="attendees-wrapper"
  >
    <ul
      className="event-attendees not-reserves"
    >
      <li
        className="event-attendee"
      >
        <img
          alt="jason carty"
          src="http://gravatar.com/avatar/c23b621863377dd53ac87b761c1e5c9a.png?s=40&d=wavatar"
        />
        <span
          className="attendee-name"
        >
          jason carty
        </span>
      </li>
    </ul>
  </div>
</div>
`;

exports[`<EventListAttendee /> matches snapshot when isReserves 1`] = `
<div
  className="event-attendees reserves"
>
  <div
    className="attendees-wrapper"
  >
    <ul
      className="event-attendees reserves"
    >
      <li
        className="event-attendee"
      >
        <img
          alt="jason carty"
          src="http://gravatar.com/avatar/c23b621863377dd53ac87b761c1e5c9a.png?s=40&d=wavatar"
        />
        <span
          className="attendee-name"
        >
          jason carty
        </span>
      </li>
    </ul>
  </div>
</div>
`;
